xchange-blockchain/src/main/java/org/knowm/xchange/blockchain/BlockchainConstants.java

```diff
@@ -11,18 +11,36 @@ public class BlockchainConstants {
     public static final String GET_FEES = "getFees";
     public static final String GET_DEPOSIT_HISTORY = "depositHistory";
     public static final String GET_WITHDRAWAL_HISTORY = "withdrawHistory";
+    public static final String GET_ORDERS = "getOrders";
+    public static final String GET_ORDER = "getOrder";
+    public static final String POST_ORDER = "postOrder";
+    public static final String CANCEL_ORDER = "cancelOrder";
+    public static final String CANCEL_ALL_ORDERS = "cancelAllOrders";
     public static final String GET_SYMBOLS = "getSymbols";
+    public static final String GET_TRADES = "getTrades";
     public static final String CURRENCY_PAIR_SYMBOL_FORMAT = "%s-%s";
     public static final String X_API_TOKEN = "X-API-Token";
     public static final String WITHDRAWAL_EXCEPTION = "Invalid WithdrawFundsParams parameter. Only DefaultWithdrawFundsParams is supported.";
     public static final String EXCEPTION_MESSAGE = "Operation failed without any error message";
     public static final String FUNDING_RECORD_TYPE_UNSUPPORTED =  "Invalid FundingRecord parameter. Only DefaultWithdrawFundsParams is supported.";
+    public static final String CURRENCY_PAIR_EXCEPTION = "Invalid TradeHistoryParams type, it should be an instance of BlockchainTradeHistoryParams";
+    public static final String ORDER_EXCEPTION = "The order was rejected probably the amount does not comply with the restrictions.  The order id is null";
```

Comment:
Your order was rejected. In most cases it happens when the amount does not comply with the restrictions.

Reply:
can we explain the restrictions?